examples/ingestion-pipeline/pipeline_lib/imageAttributes.py

```diff
@@ -0,0 +1,45 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+    https://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ """
+
+#TODO: add function that extracts color_depth and compression_ratio
+
+class ImageAttributes():
```

Comment:
Since this is really just storing data, I suggest you make it a data class, which allows to manually create the init and __str__ function: https://realpython.com/python-data-classes/

Reply:
changed it to a  data class.